src/components/LiquidityChartRangeInput/Area.tsx

```diff
@@ -37,7 +37,7 @@ export const Area = ({
             .y0(yScale(0))(
             series.filter((d) => {
               const value = xScale(xValue(d))
-              return value > 0 && value <= innerWidth
+              return value > 0 && value <= window.innerWidth
```

Comment:
Is this done by the linter? If not, I think it's better to scope this to the window for mocking.

Reply:
the rule was something like no access to globals without the prefix, this is equivalent to what it was doing before